src/main/java/io/github/cottonmc/energy/api/DefaultEnergyTypes.java

```diff
@@ -1,44 +1,51 @@
 package io.github.cottonmc.energy.api;
 
 import io.github.cottonmc.energy.CottonEnergy;
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.registry.Registry;
 
 public class DefaultEnergyTypes {
 
-	public static final EnergyType ENERGY_TYPE_EMPTY = register("empty", new EmptyEnergyType());
-
-	//TODO: values not final, discuss
-	/** Base level. Coal-powered or early-game machines. */
-	public static final EnergyType LOW_VOLTAGE = register("low_voltage", new ElectricalEnergyType(40));
-	/** Mid-level. Coal coke powered, first industrial machines */
-	public static final EnergyType MEDIUM_VOLTAGE = register("medium_voltage", new ElectricalEnergyType(80));
-	/** High-level. ??????-powered, medium-scale machines. */
-	public static final EnergyType HIGH_VOLTAGE = register("high_voltage", new ElectricalEnergyType(160));
-	/** Endgame. Uranium or higher, large-scale industrial setups or high-voltage power lines. */
-	public static final EnergyType ULTRA_HIGH_VOLTAGE = register("ultra_high_voltage", new ElectricalEnergyType(320));
-
-	public static void init() {
-
-	}
-
-	public static EnergyType register(String name, EnergyType type) {
-		Registry.register(CottonEnergy.ENERGY_REGISTRY, new Identifier("cotton", name), type);
-		return type;
-	}
-
-	public static class EmptyEnergyType implements EnergyType {
-		public EmptyEnergyType() {
-
-		}
-		@Override
-		public int getMaximumPacketSize() {
-			return 0;
-		}
-
-		@Override
-		public int getMinimumPacketSize() {
-			return 0;
-		}
-	}
+    public static final EnergyType ENERGY_TYPE_EMPTY = register("cotton:empty", new EmptyEnergyType());
+
+    //TODO: values not final, discuss
+    /** Base level. Coal-powered or early-game machines. */
+    public static final EnergyType LOW_VOLTAGE = register("low_voltage", new ElectricalEnergyType(1));
```

Comment:
We should talk over whether powers of 10 is what we wanna do here. I feel like it makes some sense but is too much of an open door for creep. That's why I had it at powers of 2 starting at 40 before.

Reply:
Values not final, discuss :)
We could instead use 1, 2, 4, 8. But then a packet from a coal coke MV furnace through a transformer only delivers two LV packets to bottom-tier machines.

Reply:
Possibly 1, 4, 16, 64? That'd be a good balance between getting a decent amount of boost every time you tier up and not throwing a thousand packets at a BE simultaneously if you want to convert from UHV down to LV.

Reply:
I would forget 1 and start at 4.

Reply:
Maybe even 16, 64, 256, 1024. Then we could have a UHV packet with 1024 WU convert down to 64 16-WU packets.

Reply:
I think either 16-1024 or 40-320 makes sense.

Reply:
16 seems extreme. I want to *start* with numbers uninflated if at all possible.

Let's say at the bottom tier, the one pulse every two to five seconds should nonetheless be capable of splitting up to four machines or so, so let's say four.

So 4, 16, 64, 256?

Reply:
Sounds good to me.

Reply:
done.